webrtc.html

```diff
@@ -11591,7 +11591,76 @@ <h3>DTMF Example</h3>
   console.log('DTMF function not available');
 }
       </pre>
-    </section><!--
+    </section>
+    <section>
+      <h3>Perfect Negotiation Example</h3>
+      <p>Perfect negotiation is a recommended pattern to abstract away signaling
+      from the rest of an application. It lets applications operate on both
```

Comment:
I think it's clear to you and I what "abstract away signaling from the rest of the application" means, but while it sounds good to abstract things away, it might not be clear what "abstracting away" means or solves unless you're already familiar with this stuff. Do we need to clarify that this means that "If the application modifies the state of the peer connection in such a way that renegotiation is required, this will happen automatically due to event listeners. In other words, if addTrack() is performed on one endpoint, steps will automatically be taken to make sure ontrack fires at the other endpoint." ?